static/src/js/components/CollectionDetails/CollectionDetailsBody.js

```diff
@@ -132,124 +133,132 @@ export const CollectionDetailsBody = ({
-                            bootstrapVariant="link"
-                            label="View All Related URLs"
-                            onClick={() => onToggleRelatedUrlsModal(true)}
-                          >
-                            View All Related URLs
-                          </Button>
-                        </>
-                      )
-                    }
-                    <a
-                      className="link link--external collection-details-body__related-link"
-                      href={urls.html.href}
-                      // eslint-disable-next-line react/jsx-no-target-blank
-                      target="_blank"
-                    >
-                      View More Info
-                    </a>
-                  </dd>
-                </>
+                doi && buildDoiLink(doi)
               }
-              {
-                temporal && (
-                <>
-                  <dt>Temporal Extent:</dt>
-                  <dd>
-                    {temporal.map((entry, i) => {
-                      const key = `temporal_entry_${i}`
-                      return <span key={key}>{entry}</span>
-                    })}
-                  </dd>
-                </>
-                )
-              }
-              <dt>GIBS Imagery Projection Availability:</dt>
-              <dd>
-                {gibsLayers && gibsLayers}
-              </dd>
-            </dl>
-          </div>
-          <div className="col collection-details-body__group--secondary">
-            <dl className="collection-details-body__info">
-              <dt>Science Keywords:</dt>
-              <dd>
+              <dl className="collection-details-body__info">
                 {
-                  scienceKeywords.length === 0 && <span>Not Available</span>
+                  <>
+                    <dt>Related URLs:</dt>
+                    <dd className="collection-details-body__related-links">
+                      {
+                        relatedUrls.length > 0 && (
+                          <>
+                            {buildRelatedUrlsList(relatedUrls)}
+                            <Button
+                              className="link link--separated collection-details-body__related-link"
+                              type="button"
+                              variant="link"
+                              bootstrapVariant="link"
+                              label="View All Related URLs"
+                              onClick={() => onToggleRelatedUrlsModal(true)}
+                            >
+                              View All Related URLs
+                            </Button>
+                          </>
+                        )
+                      }
+                      <a
+                        className="link link--external collection-details-body__related-link"
+                        href={urls.html.href}
+                        // eslint-disable-next-line react/jsx-no-target-blank
```

Comment:
I think this can be avoided by using `target="_blank" rel="noopener noreferrer"`

Reply:
Good catch